conf.yaml

```diff
@@ -1103,6 +1103,7 @@ contents:
                 repo:   apm-server
                 path:   docs
                 exclude_branches:   [ 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0 ]
+                map_branches: *mapMainToMaster
```

Comment:
You'll probably need this on the `beats` repo as well?

Reply:
I don't think so. APM Server uses a script to pull content from Beats, not the doc build.

Reply:
Oh, you mean the thing right below

Reply:
Yes

Reply:
Good catch 😅